api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -78,7 +78,7 @@ void shouldProcessClaimFromPregnancyToChildTurningFour() throws JsonProcessingEx
 
         UUID claimId = applyForHealthyStartAsPregnantWomanWithNoChildren(expectedDeliveryDate);
         Claim claim = repositoryMediator.loadClaim(claimId);
-        wiremockManager.verifyGoogleAnalyticsCalledForClaimEventWithNoChildren(claim, UPDATED_FROM_NEW_TO_ACTIVE);
+        wiremockManager.verifyGoogleAnalyticsCalledForClaimEvent(claim, UPDATED_FROM_NEW_TO_ACTIVE);
```

Comment:
Why has this changed?

Reply:
Now that we're obtaining the list of children's dob from the claim, there's no need to pass a list through to wiremock - which means that there's no need to a separate `verifyGoogleAnalyticsCalledForClaimEventWithNoChildren` method.